runtime/near-vm-runner/Cargo.toml

```diff
@@ -16,6 +16,8 @@ This crate implements the specification of the interface that Near blockchain ex
 cached = "0.12"
 wasmer-runtime = { version = "=0.17.0", features = ["default-backend-singlepass"], default-features = false }
 wasmer-runtime-core = { version = "=0.17.0" }
+wasmtime = "0.17.0"
+anyhow = "1.0.19"
```

Comment:
This crate as needed, IMO, as otherwise I cannot implement IntoVMError for anyhow::Error, and this is an error returned by wasmtime::Linker::instantiate and function runner.

Reply:
Sounds bad that wasmtime doesn't wrap and export it's own error type

Reply:
I see, my original comment is probably wrong anyway, since wasmtime pulls in this dependency anyway.